src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -91,6 +91,47 @@ private static CSharpCommandLineArguments FullParse(string commandLine, string b
             return CSharpCommandLineParser.Default.Parse(args, baseDirectory, sdkDirectory, additionalReferenceDirectories);
         }
 
+        // This test should only run when the machine's default encoding is shift-JIS
+        [ConditionalFact(typeof(HasShiftJisDefaultEncoding))]
```

Comment:
Perfect.  Conditionally running will make it easier for Matt to verify this ran correctly in the loc runs.